pandas/core/groupby/ops.py

```diff
@@ -380,7 +380,7 @@ def get_func(fname):
             # otherwise find dtype-specific version, falling back to object
             for dt in [dtype_str, 'object']:
                 f = getattr(libgroupby, "{fname}_{dtype_str}".format(
-                    fname=fname, dtype_str=dtype_str), None)
+                    fname=fname, dtype_str=dt), None)
```

Comment:
So do we even need this object fallback loop?

Reply:
not for group_add and it worked fine until now.
still it seemed like a bug.